src/test/functional/mutationServices.test/testServerUtil.ts

```diff
@@ -0,0 +1,33 @@
+import { Knex } from 'knex';
+import { ItemsEventEmitter } from '../../../businessEvents';
+import { ApolloServer } from 'apollo-server-express';
+import { buildFederatedSchema } from '@apollo/federation';
+import { typeDefs } from '../../../server/typeDefs';
+import { resolvers } from '../../../resolvers';
+import { ContextManager } from '../../../server/context';
+
+export function getServer(
+  userId: string,
+  readClient: Knex,
+  writeClient: Knex,
+  eventEmitter: ItemsEventEmitter
+) {
+  return new ApolloServer({
+    schema: buildFederatedSchema({ typeDefs, resolvers }),
+    context: () => {
```

Comment:
non-blocking, but if we're separating this out we should add the plugins to get rid of the annoying default apollo stuff

e.g. PluginInlineTraceDisabled, UsageReportingDisabled (idk exactly what they're called but similar)

Reply:
just refactored the existing code that was repeatedly creating new apollo server for every test, didnt refactor the test that was calling the `getServer` call from server.ts. 

disabled `PluginInlineTraceDisabled` , usage reporting is enabled only if we provide a `GRAPH_KEY` to the server: https://www.apollographql.com/docs/apollo-server/api/plugin/usage-reporting/ 

data from inlineTracing plugin seems to be used by gateway, do you know how enabling or disabling them affect the test server?

Reply:
If you don't explicitly disable PluginInlineTrace, then you get a console message about missing the GRAPH_KEY when it tries to send error/usage data. Since it's just about console statements that's why it's non-blocking.

Reply:
FYI - I fixed the plugins. You have to import the `Disabled` one and call it as a function to disable it (e.g. `PluginInlineTraceDisabled()`)

Reply:
thanks! added `ApolloServerPluginUsageReportingDisabled`